### ISSUE TITLE
Docker and database docker name

### ISSUE THREAD

**Issue by Merrick28** [sismics/reader#181]
Hello,

I'd like to use this with docker, but I already have a container named "db" for another database. 
Is it possible to use an environemnt variable to specify the name of the database host ?

**Comment by couladin** [comment]
You can just rename it, but if you're deploying it in a combined stack - like the `reader-distribution-docker/docker-compose.yml` config file does - there's no need to change service names. If you do change it, you'll also need to change references to it ('links' section in the web service).

I would suggest removing the `network_mode` lines, as bridged mode is rarely needed. I prefer my stacks/containers to deploy into their own subnets, and you can still access it from the IP of the device you're running Docker on.

FWIW, I just spent 15 minutes setting this up for the first time, without issues. Made a bunch of changes to the above config, deployed it with Portainer to my NAS. Trial and error, and referring to the [Compose file version 3 reference](https://docs.docker.com/compose/compose-file/compose-file-v3/), is worth the time and head-scratching. :)

**Comment by Merrick28** [comment]
Yes, I managed to make it work, but it should be easier. The user should just set an env variable  with  the database name/host/user/login, just as many other containers

**Comment by Merrick28** [comment]
My bad, I just saw the project was almost dead...

**Comment by jendib** [comment]
It's not dead by any mean. It's in a fairly stable state so quite few updates, but anyone can contribute new features (and some have).
